src/neo/SmartContract/InteropService.cs

```diff
@@ -15,6 +15,16 @@ static InteropService()
                 t.GetFields()[0].GetValue(null);
         }
 
+        public static long GetPrice(uint hash)
+        {
+            return methods[hash].GetPrice();
```

Comment:
TryGet?

Reply:
If we use TryGet, in which case the TryGet will return false? And if TryGet returns false, How to do with the GetPrice result, throw exception or return 0?

Reply:
Maybe we should change all callers, and return false in the caller if was not found

```csharp
protected override bool OnSysCall(uint method)
        {
            if (!AddGas(InteropService.GetPrice(method, CurrentContext.EvaluationStack)))
                return false;
            return InteropService.Invoke(this, method);
        }
```

Reply:
@shargon  please check my new commit, whether these change meets the requirements.